.github/workflows/main.yml

```diff
@@ -64,8 +96,6 @@ jobs:
           docker run --rm -v /var/run/docker.sock:/var/run/docker.sock --network=host testcontainers-python python diagnostics.py
       - name: Build documentation
```

Comment:
Shall we also move this one up to the `sphinx` job?

Reply:
Agreed

Reply:
resolved